test/CheckoutSdkTest/Reports/ReportsIntegrationTest.cs

```diff
@@ -52,15 +54,40 @@ private async Task ShouldGetReportDetails()
                 var reportDetails = reports.Data[0];
                 var detailsResponse = await DefaultApi.ReportsClient().GetReportDetails(reportDetails.Id);
                 
-                detailsResponse.ShouldNotBeNull();
-                detailsResponse.Id.ShouldBe(reportDetails.Id);
-                detailsResponse.CreatedOn.ShouldBe(reportDetails.CreatedOn);
-                detailsResponse.Type.ShouldBe(reportDetails.Type);
-                detailsResponse.Account.ClientId.ShouldBe(reportDetails.Account.ClientId);
-                detailsResponse.Account.EntityId.ShouldBe(reportDetails.Account.EntityId);
-                detailsResponse.From.ShouldBe(reportDetails.From);
-                detailsResponse.To.ShouldBe(reportDetails.To);
+                CheckDetailsAssertions(detailsResponse, reportDetails);
             }
         }
+        
+        [Fact]
+        private async Task ShouldGetReportFile()
+        {
+            var reports = await DefaultApi.ReportsClient().GetAllReports(_query);
+            
+            reports.ShouldNotBeNull();
+            if (!reports.Data.IsNullOrEmpty())
+            {
+                var reportDetails = reports.Data[0];
+                var detailsResponse = await DefaultApi.ReportsClient().GetReportDetails(reportDetails.Id);
+
+                CheckDetailsAssertions(detailsResponse, reportDetails);
+
+                var fileResponse = await DefaultApi.ReportsClient().GetReportFile(reportDetails.Id, reportDetails.Files[0].Id);
+
+                fileResponse.ShouldNotBeNull();
+                fileResponse.HttpStatusCode.ShouldBe(200);
+            }
+        }
+
+        private static void CheckDetailsAssertions(ReportDetailsResponse detailsResponse, ReportDetailsResponse reportDetails)

```

Comment:
Any particular reason for this method? Are we expecting to add more in the future?

Reply:
We can use it in `ShouldGetReportDetails` and `ShouldGetReportFile` tests, but we can only check in the first.

Reply:
That's fine